### PR TITLE
[RFCs] Add rfcs directory

### PR DESCRIPTION
**Pull request by vossmjp**
### Description 
Add a set of directories and sub-directories to support the oneTBB RFCs process.

Fixes # - _issue number(s) if exists_

### Type of change

_Choose one or multiple, leave empty if none of the other choices apply_

_Add a respective label(s) to PR if you have permissions_

- [ ] bug fix - _change that fixes an issue_
- [ ] new feature - _change that adds functionality_
- [ ] tests - _change in tests_
- [ ] infrastructure - _change in infrastructure and CI_
- [ X] documentation - _documentation update_

### Tests

- [ ] added - _required for new features and some bug fixes_
- [X ] not needed

### Documentation

- [ ] updated in # - _add PR number_
- [ ] needs to be updated
- [ ] not needed

### Breaks backward compatibility
- [ ] Yes
- [X ] No
- [ ] Unknown

### Notify the following users
_List users with `@` to send notifications_

### CODE REVIEW

File: rfcs/template.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+Pay close attention to the following aspects of the library:
+- API and ABI backwards compatibility. The library follows semantic versioning
+  so if any of those interfaces are to be broken, we need to state that
+  explicitly.
+- Performance implications, as performance is one of the main goals of the library.
+- Changes to the build system. While the library's primary building system is
+  CMake, there are some frameworks that may build the library directly from the sources.
+- Dependencies and support matrix: does the proposal brings any new
+  dependencies or affects the supported configurations.
```
Comment (aleksei-fedotov) [inline rfcs/template.md:40]: It is a question, right? Also, the `s` ending should be omitted in a question. Right?
```suggestion
- Dependencies and support matrix: does the proposal bring any new
  dependencies or affect the supported configurations?
```

File: rfcs/template.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+Some other common subsections here are:
+- Discussion: some people like to list all the options first (as separate
+  subsections), and then have a dedicated section with the discussion/
```
Comment (aleksei-fedotov) [inline rfcs/template.md:44]: Mistyped period?
```suggestion
  subsections), and then have a dedicated section with the discussion.
```

File: rfcs/archived/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Archived Design Documents
+
+Documents may appear in the `rfcs/archived` directory for one of 
+three reasons:
```
[the remark this answers is not in the document]
Comment (vossmjp) [reply rfcs/archived/README.md:4]: There had been three reasons, but then one was removed. I'll fix this.

File: rfcs/README.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+1. Initial RFC PR
+2. proposed
+3. experimental
+4. supported
+5. archived
+
+Most modifications or new features will naturally start as a part of a 
+Github issue or discussion. Small changes do not require a formal RFC. 
+However, if the issue or discussion leads to an idea for a modification
```
Comment (aepanchi) [inline rfcs/README.md:21]:
```suggestion
However, if the issue or discussion results in an idea for a significant change or new feature that affects the library's public API or architecture, we recommend opening a PR  to add a new RFC  to the `rfcs/proposed` directory. The RFC should provide a detailed description and design of the proposed feature.
```

File: rfcs/README.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+Github issue or discussion. Small changes do not require a formal RFC. 
+However, if the issue or discussion leads to an idea for a modification 
+or new feature that significantly impacts the library's public API or 
+architecture, it will be suggested that a PR be opened to add a new rfc 
+to the `rfcs/proposed` directory. The RFC contains a more detailed description
+and design for the feature.
+
+## General Process
+
+A template for RFCs is available as [template.md](template.md). The modified
```
Comment (aepanchi) [inline rfcs/README.md:29]:
```suggestion
A template for RFCs is available as [template.md](template.md). Place the modified
```

File: rfcs/README.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+template should be placed in subdirectory of the `rfcs/proposed` with a name
```
Comment (aepanchi) [inline rfcs/README.md:30]:
```suggestion
template in the subdirectory of the `rfcs/proposed` with a name
```

File: rfcs/README.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+of the form `<feature>_<extension_description>`. For example,
+a proposal for a new "my_op" flow graph node might be put into a directory
```
Comment (aepanchi) [inline rfcs/README.md:32]:
```suggestion
a proposal for a new ``my_op`` flow graph node should be put into the
```

File: rfcs/README.md [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+`rfcs/proposed/flow_graph_my_op_node`. The [template.md](template.md) should
```
Comment (aepanchi) [inline rfcs/README.md:33]:
```suggestion
`rfcs/proposed/flow_graph_my_op_node` directory. Use [template.md](template.md) 
```

File: rfcs/README.md [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+be used to create the `README.md` file in that directory. The folder can
```
Comment (aepanchi) [inline rfcs/README.md:34]:
```suggestion
to create the `README.md` file in that directory. The folder can 
```

File: rfcs/README.md [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+
+A template for RFCs is available as [template.md](template.md). The modified
+template should be placed in subdirectory of the `rfcs/proposed` with a name
+of the form `<feature>_<extension_description>`. For example,
+a proposal for a new "my_op" flow graph node might be put into a directory
+`rfcs/proposed/flow_graph_my_op_node`. The [template.md](template.md) should
+be used to create the `README.md` file in that directory. The folder can 
+contain other files referenced by the `README.md` file, such as figures.
+
+When two maintainers approve the PR it will be merged to the `rfcs/proposed`
```
Comment (aepanchi) [inline rfcs/README.md:37]:
```suggestion
Once two maintainers approve the PR, it is merged into the `rfcs/proposed`
```

File: rfcs/README.md [hunk 11]
```diff
@@ -0,0 +1,70 @@
...
+directory. As the RFC moves to different states, the RFC document should be
```
Comment (aepanchi) [inline rfcs/README.md:38]:
```suggestion
directory. Update the RFC document with additional information as the RFC moves to different states. 
```

File: rfcs/README.md [hunk 12]
```diff
@@ -0,0 +1,70 @@
...
+template should be placed in subdirectory of the `rfcs/proposed` with a name
+of the form `<feature>_<extension_description>`. For example,
+a proposal for a new "my_op" flow graph node might be put into a directory
+`rfcs/proposed/flow_graph_my_op_node`. The [template.md](template.md) should
+be used to create the `README.md` file in that directory. The folder can 
+contain other files referenced by the `README.md` file, such as figures.
+
+When two maintainers approve the PR it will be merged to the `rfcs/proposed`
+directory. As the RFC moves to different states, the RFC document should be 
+updated with additional information.
```
Comment (aepanchi) [inline rfcs/README.md:39]:
```suggestion

```

File: rfcs/README.md [hunk 13]
```diff
@@ -0,0 +1,70 @@
...
+
+A proposal that is subsequently implemented and released in oneTBB 
+as a preview feature will be moved to the `rfcs/experimental` folder. The
```
Comment (aepanchi) [inline rfcs/README.md:42]:
```suggestion
as a preview feature is moved into the `rfcs/experimental` folder. The
```

File: rfcs/README.md [hunk 14]
```diff
@@ -0,0 +1,70 @@
...
+RFC for a preview feature in `rfcs/experimental` should include a description
+of what is required to move from experimental to fully supported -- for example, feedback from users, demonstrated performance improvements, etc.
```
Comment (aepanchi) [inline rfcs/README.md:44]:
```suggestion
of what is required to move from experimental to fully supported. For example, user feedback, performance improvements, etc.
```

File: rfcs/README.md [hunk 15]
```diff
@@ -0,0 +1,70 @@
...
+have a link to the section in the oneTBB specification that has its 
+formal wording.
+
+A feature that is deprecated or removed or a proposal that is abandoned may be moved to the `rfcs/archived` folder.
+
+## Document Style
+
+The design documents are stored in the `rfcs` directory.
+
+- Each RFC is stored in a separate subdirectory
```
Comment (aepanchi) [inline rfcs/README.md:57]:
```suggestion

```

File: rfcs/README.md [hunk 16]
```diff
@@ -0,0 +1,70 @@
...
+  `rfcs/proposed/<feature>_<extension_description>`
```
Comment (aepanchi) [inline rfcs/README.md:58]:
```suggestion

```

File: rfcs/README.md [hunk 17]
```diff
@@ -0,0 +1,70 @@
...
+- There must be a `README.md` file that contains the main RFC itself (or links to a file that contains it in the same directory).
+  - The body of the RFC should be based on [template.md](template.md)
+  - The directory can contain other supporting files, such as images, tex formulas, and sub-proposals / sub-RFCs.
+  - It is not required to use the markdown file format based on the exact
```
Comment (aepanchi) [inline rfcs/README.md:62]:
```suggestion
  - We highly recommend using a text-based file format like markdown for easy collaboration on GitHub, but other formats like PDFs may also be acceptable.
```

File: rfcs/README.md [hunk 18]
```diff
@@ -0,0 +1,70 @@
...
+    template file for writing RFCs. However, it is strongly recommended to use
+    text-based file format that can be rendered by GitHub to allow for easy
+    collaboration using PR comments. Even so, files such as pdfs may be
+    acceptable.
+- If the RFC is written in markdown. The width of the text should be limited by
```
Comment (aepanchi) [inline rfcs/README.md:67]:
```suggestion
- For the markdown-written RFC, keep the text width within
```

File: rfcs/README.md [hunk 19]
```diff
@@ -0,0 +1,70 @@
...
+  80 symbols, unless there is a need to violate this rule, e.g. because of
```
Comment (aepanchi) [inline rfcs/README.md:68]:
```suggestion
  80 characters, unless there is a reason to violate this rule, e.g., 
```

File: rfcs/README.md [hunk 20]
```diff
@@ -0,0 +1,70 @@
...
+  long links or wide tables.
+- It is also recommended to read through existing RFCs to better understand the general writing style and required elements.
```
Comment (aepanchi) [inline rfcs/README.md:70]:
```suggestion
- Review existing RFCs to understand the expected writing style and required elements.
```

File: rfcs/archived/README.md [hunk 21]
```diff
@@ -0,0 +1,13 @@
...
+
+1. The document describes a feature or extension that has been deprecated and then removed.
```
Comment (aepanchi) [inline rfcs/archived/README.md:6]:
```suggestion
1. The document describes a deprecated or removed feature/extension.
```

File: rfcs/archived/README.md [hunk 22]
```diff
@@ -0,0 +1,13 @@
...
+2. The document describes a feature or extension that was proposed but did
```
Comment (aepanchi) [inline rfcs/archived/README.md:7]:
```suggestion
2. The document describes a proposed feature or extension that have
```

File: rfcs/archived/README.md [hunk 23]
```diff
@@ -0,0 +1,13 @@
...
+not (ultimately) become a fully support feature. 
+
+Design documents that appear in the `rfcs/archived` folder should include a 
+section that describes why the document has been archived. Documents may
```
Comment (aepanchi) [inline rfcs/archived/README.md:11]:
```suggestion
reason for archiving.  Documents may 
```

File: rfcs/archived/README.md [hunk 24]
```diff
@@ -0,0 +1,13 @@
+# Archived Design Documents
+
+Documents may appear in the `rfcs/archived` directory for one of 
+three reasons:
+
+1. The document describes a feature or extension that has been deprecated and then removed.
+2. The document describes a feature or extension that was proposed but did
+not (ultimately) become a fully support feature. 
+
+Design documents that appear in the `rfcs/archived` folder should include a
```
Comment (aepanchi) [inline rfcs/archived/README.md:10]:
```suggestion
Design documents that appear in the `rfcs/archived` folder should describe a 
```

File: rfcs/experimental/README.md [hunk 25]
```diff
@@ -0,0 +1,28 @@
+# Design documents for experimental features
```
Comment (aepanchi) [inline rfcs/experimental/README.md:1]:
```suggestion
# Design Documents for Experimental Features
```

File: rfcs/experimental/README.md [hunk 26]
```diff
@@ -0,0 +1,28 @@
...
+
+Experimental proposals describe extensions that have been implemented and
```
Comment (aepanchi) [inline rfcs/experimental/README.md:3]:
```suggestion
Experimental proposals describe extensions that are implemented and
```

File: rfcs/experimental/README.md [hunk 27]
```diff
@@ -0,0 +1,28 @@
...
+have been released as preview features in the oneTBB library. A preview
```
Comment (aepanchi) [inline rfcs/experimental/README.md:4]:
```suggestion
released as preview features in the oneTBB library. A preview
```

File: rfcs/experimental/README.md [hunk 28]
```diff
@@ -0,0 +1,28 @@
...
+feature is expected to have an implementation that is of comparable quality
+to a fully supported feature. It should also have sufficient tests.
```
Comment (aepanchi) [inline rfcs/experimental/README.md:6]:
```suggestion
to a fully supported feature. Sufficient tests are required.
```

File: rfcs/experimental/README.md [hunk 29]
```diff
@@ -0,0 +1,28 @@
...
+
+An experimental feature does not yet appear as part of the oneTBB 
+specification and therefore its interface and design can still change.
```
Comment (aepanchi) [inline rfcs/experimental/README.md:9]:
```suggestion
specification. Therefore, the interface and design can change.
```

File: rfcs/experimental/README.md [hunk 30]
```diff
@@ -0,0 +1,28 @@
...
+There is no commitment to backwards compatibility for a preview
```
Comment (aepanchi) [inline rfcs/experimental/README.md:10]:
```suggestion
There is no commitment to backward compatibility for a preview
```

File: rfcs/experimental/README.md [hunk 31]
```diff
@@ -0,0 +1,28 @@
...
+feature.
+
+Since documents in this directory describe preview features, they
```
Comment (aepanchi) [inline rfcs/experimental/README.md:13]:
```suggestion
The documents in this directory 
```

File: rfcs/experimental/README.md [hunk 32]
```diff
@@ -0,0 +1,28 @@
...
+should include a list of the exit conditions that need to be met to move from
+preview to fully supported. These conditions might include demonstrated
+performance improvements, demonstrated interest from the community,
+acceptance of the required oneTBB specification changes, etc. 
+
+For features that require oneTBB specification changes, the document might
+include wording for those changes or a link to any PRs that have been opened
```
Comment (aepanchi) [inline rfcs/experimental/README.md:20]:
```suggestion
include wording for those changes or a link to any PRs that opened
```

File: rfcs/experimental/README.md [hunk 33]
```diff
@@ -0,0 +1,28 @@
...
+against the specification.
+
+Proposals should not remain in the experimental directory forever. The 
+expectation is that the proposals will eventually move either to the
```
Comment (aepanchi) [inline rfcs/experimental/README.md:24]:
```suggestion
It should move either to the
```

File: rfcs/experimental/README.md [hunk 34]
```diff
@@ -0,0 +1,28 @@
...
+supported folder when they become fully supported or to the archived
```
Comment (aepanchi) [inline rfcs/experimental/README.md:25]:
```suggestion
supported folder when they become fully supported or the archived 
```

File: rfcs/proposed/README.md [hunk 35]
```diff
@@ -0,0 +1,10 @@
+# Design documents for proposed features
```
Comment (aepanchi) [inline rfcs/proposed/README.md:1]:
```suggestion
# Design Documents for Proposed Features
```

File: rfcs/proposed/README.md [hunk 36]
```diff
@@ -0,0 +1,10 @@
...
+
+Proposed features have reached some level of consensus within the
```
Comment (aepanchi) [inline rfcs/proposed/README.md:3]:
```suggestion
Proposed features in this directory have reached some level of consensus within the 
```

File: rfcs/proposed/README.md [hunk 37]
```diff
@@ -0,0 +1,10 @@
+# Design documents for proposed features
+
+Proposed features have reached some level of consensus within the 
+community. A proposal appears in this directory because the
```
Comment (aepanchi) [inline rfcs/proposed/README.md:4]:
```suggestion
community, indicating that they have potential and deserve further development. 
```

File: rfcs/proposed/README.md [hunk 38]
```diff
@@ -0,0 +1,10 @@
...
+community believes it has merit and further work should be done.
```
Comment (aepanchi) [inline rfcs/proposed/README.md:5]:
```suggestion

```

File: rfcs/proposed/README.md [hunk 39]
```diff
@@ -0,0 +1,10 @@
...
+However, the proposed changes have not yet been released as a
+preview or fully supported feature of the library.
+
+RFCs in the `rfcs/proposed` directory, should describe the motivation,
```
Comment (aepanchi) [inline rfcs/proposed/README.md:9]:
```suggestion
RFCs in the `rfcs/proposed` directory should explain the motivation,
```

File: rfcs/proposed/README.md [hunk 40]
```diff
@@ -0,0 +1,10 @@
...
+design and open questions related to the proposed extension.
```
Comment (aepanchi) [inline rfcs/proposed/README.md:10]:
```suggestion
design, and open questions related to the proposed extension.
```

File: rfcs/supported/README.md [hunk 41]
```diff
@@ -0,0 +1,11 @@
+# Design documents for supported features
```
Comment (aepanchi) [inline rfcs/supported/README.md:1]:
```suggestion
# Design Documents for Supported Features
```

File: rfcs/supported/README.md [hunk 42]
```diff
@@ -0,0 +1,11 @@
...
+
+Supported proposals describe extensions that have been implemented and
```
Comment (aepanchi) [inline rfcs/supported/README.md:3]:
```suggestion
Supported proposals describe extensions implemented and
```

File: rfcs/supported/README.md [hunk 43]
```diff
@@ -0,0 +1,11 @@
...
+released as fully supported features of the oneTBB library. A fully supported
+feature has a high-quality implementation. If the proposal impacted the
+public API of the library it should appear in the oneTBB specification and
```
Comment (aepanchi) [inline rfcs/supported/README.md:6]:
```suggestion
public API of the library, it should appear in the oneTBB specification and
```

File: rfcs/template.md [hunk 44]
```diff
@@ -0,0 +1,68 @@
+# Descriptive name for the proposal
```
Comment (aepanchi) [inline rfcs/template.md:1]:
```suggestion
# Descriptive Name for the Proposal
```

File: rfcs/template.md [hunk 45]
```diff
@@ -0,0 +1,68 @@
...
+- Performance improvements with the data, if available.
+- Improved engineering practices.
+
+Introduction may also include any additional information that sheds light on
+the proposal, such as history of the matter, links to relevant issues and
+discussions, etc.
+
+## Proposal
+
+A full and detailed description of the proposal, with highlighted consequences.
```
Comment (aepanchi) [inline rfcs/template.md:19]:
```suggestion
A full and detailed description of the proposal with highlighted consequences.
```

File: rfcs/template.md [hunk 46]
```diff
@@ -0,0 +1,68 @@
...
+
+Depending on the kind of the proposal, the description should cover:
+
+- New use cases that are supported by the extension.
```
Comment (aepanchi) [inline rfcs/template.md:23]:
```suggestion
- New use cases supported by the extension.
```

File: rfcs/template.md [hunk 47]
```diff
@@ -0,0 +1,68 @@
...
+- The expected performance benefit for a modification. 
+- The interface of extensions including class definitions or function 
+declarations.
+
+A proposal should include the alternatives that were considered with listed
```
Comment (aepanchi) [inline rfcs/template.md:28]: A proposal should clearly outline the alternatives that were considered, along with their pros and cons. Each alternative should be clearly separated to make discussions easier to follow.

File: rfcs/template.md [hunk 48]
```diff
@@ -0,0 +1,68 @@
...
+pros and cons. The alternatives should be clearly separated to make possible
+discussions clear.
+
+Pay close attention to the following aspects of the library:
+- API and ABI backwards compatibility. The library follows semantic versioning
```
Comment (aepanchi) [inline rfcs/template.md:33]:
```suggestion
- API and ABI backward compatibility. The library follows semantic versioning
```

File: rfcs/template.md [hunk 49]
```diff
@@ -0,0 +1,68 @@
...
+- Performance implications, as performance is one of the main goals of the library.
+- Changes to the build system. While the library's primary building system is
+  CMake, there are some frameworks that may build the library directly from the sources.
+- Dependencies and support matrix: does the proposal bring any new
+  dependencies or affect the supported configurations?
+
+Some other common subsections here are:
+- Discussion: some people like to list all the options first (as separate
+  subsections), and then have a dedicated section with the discussion.
+- Listing of the proposed API and examples of its usage.
```
Comment (aepanchi) [inline rfcs/template.md:45]:
```suggestion
- List of the proposed API and examples of its usage.
```

File: rfcs/template.md [hunk 50]
```diff
@@ -0,0 +1,68 @@
...
+- Testing aspects.
+- Short explanation and links to the related sub-proposals, if any. Such
+  sub-proposals could be organized as separate standalone RFCs, but this is
+  not mandatory. If the changes is insignificant, or doesn't make any sense
```
Comment (aepanchi) [inline rfcs/template.md:49]:
```suggestion
  not mandatory. If the change is insignificant or doesn't make any sense
```

File: rfcs/template.md [hunk 51]
```diff
@@ -0,0 +1,68 @@
...
+  without the original proposal it is fine to have it in the RFC.
```
Comment (aepanchi) [inline rfcs/template.md:50]:
```suggestion
  without the original proposal, you can have it in the RFC.
```

File: rfcs/template.md [hunk 52]
```diff
@@ -0,0 +1,68 @@
...
+- Execution plan if approved, aka next steps.
```
Comment (aepanchi) [inline rfcs/template.md:51]:
```suggestion
- Execution plan (next steps), if approved.
```

**Comment by vbm23** [comment]
Requesting @akukanov and @pavelkumbrasev for pending reviews

File: rfcs/README.md [hunk 53]
```diff
@@ -0,0 +1,66 @@
...
+contain other files referenced by the `README.md` file, such as figures.
+
+When two maintainers approve the PR it will be merged to the `rfcs/proposed`
+directory. As the RFC moves to different states, the RFC document should be 
+updated with additional information.
+
+A proposal that is subsequently implemented and released in oneTBB 
+as a preview feature will be moved to the `rfcs/experimental` folder. The
+RFC for a preview feature in `rfcs/experimental` should include a description
+of what is required to move from experimental to fully supported -- for example, feedback from users, demonstrated performance improvements, etc.
```
[the remark this answers is not in the document]
Comment (vossmjp) [reply rfcs/README.md]: I believe that I have addressed all line lengths now.

**Comment by vbm23** [comment]
@akukanov Requesting for reviewing so this can be merged

File: rfcs/README.md [hunk 54]
```diff
@@ -0,0 +1,81 @@
...
+2. Proposed
+3. Experimental
+4. Supported
+5. Archived
+
+Most modifications or new features will naturally start as a part of a 
+GitHub issue or discussion. Small changes do not require a formal RFC. 
+However, if the issue or discussion results in an idea for a significant 
+change or new feature that affects the library's public API or architecture, 
+we recommend opening a PR  to add a new RFC  to the `rfcs/proposed` directory.
```
Comment (pavelkumbrasev) [inline rfcs/README.md:25]:
```suggestion
we recommend opening a PR to add a new RFC to the `rfcs/proposed` directory. 
```

File: rfcs/experimental/README.md [hunk 55]
```diff
@@ -0,0 +1,28 @@
...
+preview to fully supported. These conditions might include demonstrated
+performance improvements, demonstrated interest from the community,
+acceptance of the required oneTBB specification changes, etc. 
+
+For features that require oneTBB specification changes, the document might
+include wording for those changes or a link to any PRs that opened
+against the specification.
+
+Proposals should not remain in the experimental directory forever. The 
+It should move either to the
```
Comment (pavelkumbrasev) [inline rfcs/experimental/README.md:24]:
```suggestion
Proposals should not remain in the experimental directory forever.
It should move either to the
```

**Review by pavelkumbrasev — COMMENTED** [review]
Other than that LGTM!

File: rfcs/README.md [hunk 56]
```diff
@@ -0,0 +1,81 @@
...
+links to a file that contains it in the same directory).
+  - The RFC should follow the [template.md](template.md) structure. 
+  - The directory can contain other supporting files, such as images, tex 
+  formulas, and sub-proposals / sub-RFCs.
+  - We highly recommend using a text-based file format like markdown for easy 
+  collaboration on GitHub, but other formats like PDFs may also be acceptable.
+    template file for writing RFCs. However, it is strongly recommended to use
+    text-based file format that can be rendered by GitHub to allow for easy
+    collaboration using PR comments. Even so, files such as pdfs may be
+    acceptable.
```
Comment (akukanov) [inline rfcs/README.md:76]: Looks like there are two variants of wording, one of which is perhaps an accidental artifact.

File: rfcs/README.md [hunk 57]
```diff
@@ -0,0 +1,81 @@
...
+- For the markdown-written RFC, keep the text width within
+  80 characters, unless there is a reason to violate this rule, e.g., 
+  long links or wide tables.
```
Comment (akukanov) [inline rfcs/README.md:79]: Are there reasons for 80 chars width limit? IIRC our informal guidelines for source files allow 100-120 characters per line.

File: rfcs/template.md [hunk 58]
```diff
@@ -0,0 +1,68 @@
...
+open questions.
+
+For proposals released as preview features that are in the `rfcs/experimental`
+directory, list the exit conditions to move from preview to fully supported.
+These conditions might include demonstrated performance improvements,
+acceptance of specification changes, etc.
+
+For proposals in the `rfcs/supported` directory, provide a link to the
+any section(s) in the oneTBB specification that related to the proposal.
+For modifications that do not affect the public API, no link is needed.
```
Comment (akukanov) [inline rfcs/template.md:68]: I am not sure this information is useful in the template. The reason is that I'd expect only the initial state of the proposal to use the template, while all later stages would modify an existing RFC, and not the template.

So I would change this to "Open questions" or alike, and remove the rest which essentially duplicates information already described in readme files.

File: rfcs/template.md [hunk 59]
```diff
@@ -0,0 +1,68 @@
...
+
+The motivation could be:
+- Improved users experience for API changes and extensions. Code snippets to
+  showcase the benefits would be nice here.
+- Performance improvements with the data, if available.
+- Improved engineering practices.
+
+Introduction may also include any additional information that sheds light on
+the proposal, such as history of the matter, links to relevant issues and
+discussions, etc.
```
Comment (akukanov) [inline rfcs/template.md:15]: Would it make sense to format the boilerplate text int the template to be italicized and maybe put into some brackets?

**Comment by vbm23** [comment]
@pavelkumbrasev Can you help address open review comments from @akukanov thereby enabling approval

File: rfcs/README.md (hunk 57 above)
Comment (vossmjp) [reply rfcs/README.md:79]: I believe this was carried over as a rule from the other oneAPI repo we looked at as an example for RFCs. I can bump it to 100 for oneTBB, I have no strong preference here.

File: rfcs/template.md (hunk 58 above)
Comment (vossmjp) [reply rfcs/template.md:68]: I will rename the section "Open Questions". I think the info might be helpful for someone new to the process, even though you are right that they will likely delete the suggested for later steps when writing the original proposal. For now, I'll keep the extra info, but just rename the section.

File: rfcs/template.md (hunk 59 above)
Comment (vossmjp) [reply rfcs/template.md:15]: Looking at the rendered text, I think it's clear without brackets. But we can revisit later if people are confused.

**Review by pavelkumbrasev — APPROVED** [review]
LGTM!

**Pull request merged**